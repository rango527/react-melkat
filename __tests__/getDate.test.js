import formatDate from '../utils/formatDate'

describe('util - formatDate', () => {
  it('should fail gracefully when there is no date for some reason', () => {
    expect(formatDate('')).toMatchSnapshot()
  })
})
