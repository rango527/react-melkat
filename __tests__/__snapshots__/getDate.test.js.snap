// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`util - formatDate should fail gracefully when there is no date for some reason 1`] = `""`;
